pkg/infra/client_test.go

```diff
@@ -8,21 +8,21 @@ import (
 
 var _ = Describe("Client", func() {
 
-	Context("Should Error handle", func() {
+	Context("Client Error handling", func() {
```

Comment:
in sequence of `describe`, `context`, `it` right?
`Client`, `Client Error handling`, `captures error from xxx`?
or 
`Client`, `should`, `handle error from xxx`